I don't know why it is disable, but it works better for me & video
on chrome9 in HP2133
*** a/src/via_dri.c	2010-05-06 21:28:29.000000000 +0300
--- b/src/via_dri.c	2010-06-11 16:49:08.000000000 +0300
*************** VIADRIScreenInit(ScreenPtr pScreen)
*** 598,604 ****
          case VIA_P4M900:
          case VIA_VX800:
          case VIA_VX855:
!             pDRIInfo->clientDriverName = "swrast";
              break;
          default:
              pDRIInfo->clientDriverName = VIAClientDriverName;
--- 598,604 ----
          case VIA_P4M900:
          case VIA_VX800:
          case VIA_VX855:
!             pDRIInfo->clientDriverName = "unichrome";
              break;
          default:
              pDRIInfo->clientDriverName = VIAClientDriverName;
